src/stores/room-list/previews/MessageEventPreview.ts

```diff
@@ -55,7 +55,10 @@ export class MessageEventPreview implements IPreview {
         }
 
         if (hasHtml) {
-            body = sanitizedHtmlNodeInnerText(body);
+            body = sanitizeHtml(body, {
+                // only show strikethrough and bold in message previews
+                allowedTags: ['del', 'strong'],
```

Comment:
Is there any reason not to allow italics and code?

Reply:
No there isn't!

I'm not sure about code. But adding `<i>` and `<u>` tags should allow for italics and underlined text in message previews.
So, depends on what tags we want to retain, I guess.

Reply:
`sanitize-html` has [docs](https://www.npmjs.com/package/sanitize-html) for allowed tags. It looks like `<code>` is what you're looking for. If you could test this and add the previously mentioned tags it would be awesome!

Reply:
Okay, I just tested for code. (the <pre> tag basically)
Looks like it takes more height and messes up with height calculation. So, some text goes hidden.

Reply:
Yeah, the `<pre>` tag won't work but I _think_ you could get a way with to `<code>`

Reply:
> 
> 
> `sanitize-html` has [docs](https://www.npmjs.com/package/sanitize-html) for allowed tags. It looks like `<code>` is what you're looking for. If you could test this and add the previously mentioned tags it would be awesome!

Sorry, I kinda missed this reply. Just saw it. Yeah, `<code>` tag should give similar results.

Reply:
O

> 
> 
> Yeah, the `<pre>` tag won't work but I _think_ you could get away with `<code>`

Okay, again seeing this kinda late. Maybe GitHub wants me to hard refresh?
But, yeah works like a charm!

So, I guess I should change the allowed tags list to:
`u, i, strong, del, code`?

Reply:
> Maybe GitHub wants me to hard refresh?

Yeah, it's acting pretty weird sometimes...

> So, I guess I should change the allowed tags list to:
`u, i, strong, del, code`?

Yeah, that sounds sane!